pkgs/development/libraries/lirc/default.nix

```diff
@@ -42,6 +42,7 @@ stdenv.mkDerivation rec {
     "--with-systemdsystemunitdir=$(out)/lib/systemd/system"
     "--enable-uinput" # explicit activation because build env has no uinput
     "--enable-devinput" # explicit activation because build env has no /dev/input
+    "--with-lockdir=/run/lirc/lock" # /run/lock is not writable for 'lirc' user
```

Comment:
The lock dir is a decision of the operating system configuration, so please move this line to NixOS systemd.servcies.lircd.ExecStart

Reply:
I'd like to do that, but the path is compiled into the software (see [serial.c#L21](https://github.com/queezle42/lirc/blob/master/lib/serial.c#L21)).

Reply:
``/var/run`` should be a symlink to ``/run``. If someone really needs to change this we can make this overridable in the future.